docs/04-reference/winglang-spec.md

```diff
@@ -2062,11 +2064,52 @@ let bucket = cdk.aws_s3.Bucket(
 );
 ```
 
-### 5.2 Internal Libraries
+### 5.1.2 Internal Libraries
 
 Wing libraries themselves are JSII modules. They can be used in all other JSII
 supported languages.
 
+## 5.2 JavaScript
+
+The `extern(file)` modifier can be used on method declarations (in classes and resources) to indicate that a method is backed by an implementation imported from a JavaScript file.
+
+In the following example, the static inflight method `make_id` is implemented
+in `helper.js`:
+
+```js
+// task-list.w
+resource TaskList {
+  // ...
+
+  inflight add_task(title: str) {
+    let id = TaskList.uuid();
+    this.bucket.put(id, title);
+  }
+
+  extern("./helpers.js") static inflight make_id(): str;
+} 
+
+// helpers.js
+const uuid = require('uuid');
+
+exports.make_id = function() {
```

Comment:
I was expecting to see the camelCase version here
```js
exports.makeId = function() {
```

Reply:
I wouldn't over complicate this. Let's keep it 1:1.